microsoft-edge/extensions-chromium/getting-started/picture-inserter-content-script.md

```diff
@@ -406,32 +420,26 @@ The sample creates and inject the content page that's embedded on the current ac
 ```
 
 
-<!-- ------------------------------ -->
-#### The manifest's `content_scripts` section injects JavaScript into all browser tabs
+<!-- ====================================================================== -->
+## The manifest's `content_scripts` section injects JavaScript into all browser tabs
 
 In the `content_scripts` key section of the manifest file, the `matches` attribute is set to `<all_urls>`, which means that `.js` (or `.css`) files that are specified in the `content_scripts` key section are injected into all browser tab pages when each tab is loaded.  The allowed files types that can be injected are JavaScript (`.js`) and CSS (`.css`).
 
-In this sample, the following files are specified:
-* `content-scripts/content.js`
-* `lib/jquery.min.js`
-
+This sample's manifest specifies a single content script, `content-scripts/content.js`.
 
-<!-- ------------------------------ -->
-#### How jQuery support is added (`jquery.min.js`)
-
-`jquery.min.js` is a predefined, minified file to support jQuery expressions starting with `$` in `content.js`.
-
-In injected content scripts, it's common to use jQuery (`$`).  This sample includes a minified version of jQuery, residing in the extension package as `lib\jquery.min.js`.
-
-The content script (`content.js`) runs in its own sandbox, which means that the jQuery that's injected into `content.js` isn't shared with the current webpage.
 
+<!-- ====================================================================== -->
+## Each content script runs in its own thread, sandboxed
 
-<!-- ------------------------------ -->
-#### Understanding the thread
+Each tab page (and extension) runs in its own thread.  Even if the browser tab contains a webpage that runs JavaScript code, the `content.js` script that's injected in that browser tab by the extension doesn't have access to that JavaScript code.  The injected content script only has access to the DOM of the webpage.
 
-Each tab page (and extension) runs in its own thread.
+The content script (`content.js`) runs in its own sandbox, separate from any `.js` file that's used by the current webpage, such as `to-do.js` when the current page is `/demo-to-do/index.html`.  The [index.html source file](https://github.com/MicrosoftEdge/Demos/blob/main/demo-to-do/index.html) loads `to-do.js` as follows:
 
-Even if the browser tab contains a webpage that runs JavaScript code, the `content.js` script that's injected in that browser tab by the extension doesn't have access to that JavaScript code.  The injected content script only has access to the DOM of the webpage.
+```html
+    <script src="to-do.js"></script>
+  </body>
+</html>
```

Comment:
I don't think this paragraph, and the code snippet, are very useful here. It doesn't matter how the page loads its own JS files. The point is that the extension's content script doesn't have to it. And I feel like the first paragraph of this h2 section already says this.